test/.gitignore

```diff
@@ -1 +1,2 @@
 !*.zip
+/0/
```

Comment:
Created by what? I have never seen such a file. I think something may be broken on your machine.